cfsan-snp-pipeline/2.2.1/Dockerfile

```diff
@@ -0,0 +1,121 @@
+FROM ubuntu:xenial as app
+
+# metadata
+LABEL base.image="ubuntu:xenial"
+LABEL dockerfile.version="1"
+LABEL software="cfsan_snp_pipeline"
+LABEL software.version="2.2.1"
+LABEL description="A pipeline for the production of SNP matrices from sequence data used in the phylogenetic analysis of pathogenic organisms sequenced from samples of interest to food safety."
+LABEL website="https://github.com/CFSAN-Biostatistics/snp-pipeline"
+LABEL license="https://github.com/CFSAN-Biostatistics/snp-pipeline/blob/master/LICENSE.txt"
+LABEL original.maintainer="Justin Payne"
+LABEL original.maintainer.email="justin.payne@fda.hhs.gov"
+LABEL maintainer="Curtis Kapsak"
+LABEL maintainer.email="pjx8@cdc.gov"
+
+# This Dockerfile is based on a Docker file from Justin Payne which can be found here:
+# https://github.com/CFSAN-Biostatistics/snp-pipeline/blob/master/Dockerfile
+# The dockerfile has been modified from it's original form by Curtis Kapsak and Sam Baird
+
+WORKDIR /tmp/
+RUN apt-get update \
+	&& apt-get install -y --no-install-recommends \
+		build-essential \
+		bzip2 \
+		default-jre \
+		g++ \
+		gcc \
+		git \
+		gsl-bin \
+		libgsl0-dev \
+		libtbb-dev \
+		libbz2-dev \
+		liblzma-dev \
+		make \
+		ncurses-dev \
+		python \
+		python-dev \
+		tabix \
+		wget \
+		zlib1g-dev \
+        ca-certificates \
+	&& apt-get clean
```

Comment:
This is looking good!

I think ended with `apt-get clean` is probably fine, but I do recommend ending with `apt-get autoclean && rm -rf /var/lib/apt/lists/*` because this removes slightly more unneeded files. This isn't a required change, though.

From the apt-get man page:

> clean: clean clears out the local repository of retrieved package files. It removes everything but the lock file from /var/cache/apt/archives/ and /var/cache/apt/archives/partial/. When APT is used as a dselect(1) method, clean is run automatically. Those who do not use dselect will likely want to run apt-get clean from time to time to free up disk space.

> autoclean: Like clean, autoclean clears out the local repository of retrieved package files. The difference is that it only removes package files that can no longer be downloaded, and are largely useless. This allows a cache to be maintained over a long period without it growing out of control. The configuration option APT::Clean-Installed will prevent installed packages from being erased if it is set to off.